client/scss/layouts/_home.scss

```diff
@@ -7,4 +7,46 @@
     // pull out the icon so it aligns with no-icon text
     margin-inline-start: -1.75em;
   }
+
+  .w-header {
+    @include nice-padding();
+    padding-top: theme('spacing.16');
+
+    @include media-breakpoint-up(sm) {
+      .w-header__title {
+        // Font size only to be used on the home header
+        font-size: 3.125rem;
+      }
+    }
+
+    .row {
+      padding-inline-start: 4.6875rem; // 75rem
+
+      @include media-breakpoint-up(sm) {
+        padding-inline-start: 6.56rem; // 105px
+      }
+    }
+
+    .w-header__glyph.avatar {
+      margin-inline-start: calc(0 - theme('spacing.5'));
+      margin-top: 0;
+
+      @include media-breakpoint-up(sm) {
+        margin-inline-start: -3.75rem;
+      }
+    }
+
+    .w-panel-upgrade-notification {
+      padding: 0;
+    }
+  }
+
```

Comment:
I know that the dashboard/ home is a bit unique but if at all possible it would be great if we can find a way to avoid bespoke styles for this page's header. 

The goal of the unification work was to simplify the headers and remove a heap of overrides on different pages. 

It might be good to point out that this page is the only one that uses the avatar, so you can kind of treat that as the styling hook. 

Instead of overriding this pages header, you adjust the base shared header styles to look like we want whenever there is an avatar.

Reply:
This makes sense! I've made a revision of a slightly different approach. I've added a new component for header_with_avatar.html just because with the regular header.html it has many nested elements with styling that needs to be overwritten to make this header look correct. 

Since this template is likely only ever going to be used on the homepage it would make a good tailwind utility candidate too. This is all the code we would really need to basically build it that way (no external css):

``` HTML
<header class="nice-padding w-pt-20 sm:w-pt-16 w-flex w-flex-row w-items-start w-gap-5">
    <div class="avatar sm:w-w-20 sm:w-h-20 w-flex-shrink-0"><img src="{{ avatar }}" alt="" /></div>
    <div>
        <h1 id="header-title" class="w-h1 md:w-text-[3.125rem] w-my-0 w-font-extrabold w-leading-none">{{ header_title }}</h1>
        <span class="w-body-text-large w-text-primary">{{ user|user_display_name }}</span>
    </div>
</header>
```

Anyway i've moved these styles into the `header.scss` in the more verbose format using `theme()`.

I've also noticed avatar was only actually ever used on the home page so with this new template we don't need it as a param inside header.html so i've stripped that out. 

Curious to know your thoughts